.github/workflows/clean-aws-resources.yml

```diff
@@ -94,4 +94,37 @@ jobs:
           aws-region: ${{ matrix.region }}
 
       - name: Clean old dedicated host
-        run: go run ./tool/clean/clean_dedicated_host/clean_dedicated_host.go --tags=clean
\ No newline at end of file
+        run: go run ./tool/clean/clean_dedicated_host/clean_dedicated_host.go --tags=clean
+
+  clean-internal-hosts:
```

Comment:
It would be better to have a general clean: e.g clean external hosts. We could add a matrix to change between external and internal assume role.

Reply:
The external does not cleaning at least not from what I have seen

Reply:
Its the same case with ADOT. We won't know. However, we don't need to add it now but only changing  the name to `clean-EC2-host` and also add a row for internal only